src/components/vacation-requests-table/vacation-requests-table-toolbar/toolbar-form/toolbar-form.tsx

```diff
@@ -117,6 +115,13 @@ const ToolbarForm = ({
     }
   }, [toolbarFormMode]);
 
+  // Set dateTimeTomorrow based on vacation type
+  const isSicknessOrChildSickness =
+    vacationData.type === VacationType.SICKNESS || vacationData.type === VacationType.CHILD_SICKNESS;
+    const dateTimeTomorrow = isSicknessOrChildSickness
```

Comment:
Un-indent

Reply:
This dateTimeTomorrow could be moved to where dateTimeTomorrow was previously initialized, line 44 above the defaultDateRange. defaultDateRnage could then also use dateTimeTomorrow rather than the inline DateTime.now().plus({ days: 1})

Reply:
if I move dateTimeTomorrow to where dateTimeTomorrow was previously initialized it will cause error Block-scoped variable 'isSicknessOrChildSickness' used before its declaration.